### PR TITLE
Julfixes

### PR DESCRIPTION
**Pull request by dshanske**
This enables webmentions by default on the theory if you enable the plugin, you must want them, so it shouldn't be triggered by the pingback flag setting.

Also fixes the auto approve whitelist and a few other minor tweaks.

### CODE REVIEW

**Comment by pfefferle** [comment]
@dshanske it seems that you have made your changes to an old revision and that caused some conflicts.

File: includes/class-webmention-admin.php [hunk 1]
```diff
@@ -23,8 +23,6 @@ public static function init() {
 		add_filter( 'manage_edit-comments_columns', array( 'Webmention_Admin', 'comment_columns' ) );
 		add_filter( 'manage_comments_custom_column', array( 'Webmention_Admin', 'manage_comments_custom_column' ), 10, 2 );
 
-		// Webmention whitelist
```
Comment (pfefferle) [inline includes/class-webmention-admin.php]: oops... thanks for fixing my c&p issue!

**Comment by dshanske** [comment]
I will see about rebasing

**Comment by dshanske** [comment]
Should be fixed now. Sorry.

**Comment by pfefferle** [comment]
Thanks :)

**Pull request merged**